### ISSUE TITLE
Update (or remove) Bandcamp player

### ISSUE THREAD

**Issue by bhrutledge** [roperi/myband#4]
It looks like the player on https://coctelduo.com/music/solaz/ isn't for the album that's on that page.

**Comment by roperi** [comment]
Correct. I used this for illustration purposes while I'm training the band on how to use the admin interface.

**Comment by bhrutledge** [comment]
Cool. I'm looking forward to hearing an album someday. :)

**Comment by roperi** [comment]
Me too!
